src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: YouTube enhances UX with Media Capabilities API
+subhead: |
+  Ensure buttery smooth playback when your users are on different devices.
+authors:
+  - beaufortfrancois
+date: 2019-06-05
+hero: hero.jpeg
+hero_position: top
+alt: Youtube application screengrab
+description: |
+  In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase
+  in MTBR with only a 0.4% decrease in average resolution of videos served.
+tags:
+  - post
+  - media
+  - ux
+  - case-study
+---
+
+In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase in
+MTBR with only a 0.4% decrease in average resolution of videos served.
+
+{% Aside 'key-term' %}
+MTBR (Mean Time Between Rebuffers) is total play time divided by the number of
+rebuffering events.
+{% endAside %}
+
+## The Problem
+
+Typically, media sites will have several variants that they can present users,
+encoded in different frame rates, resolutions, and codecs. Until recently, while
+choosing which variant to play for an individual user, web developers had to
+rely solely on `isTypeSupported()` or `canPlayType()` to determine whether each
+variant could be played in that user’s browser. While this told the developer
+whether media could be played at all, it didn’t provide an indication of
+playback quality, such as whether the media playback would experience frame
+drops or rapidly drain the device’s battery. In the absence of this signal,
+developers either had to create their own heuristics, or just assume that if a
+device could playback a codec/resolution combination, it could do so smoothly
+and power efficiently. For users with less capable devices, this often led to a
+very poor experience.
+
+## The Solution
+
+The [Media Capabilities](https://wicg.github.io/media-capabilities/) API allows
+websites to get more information about the client’s video decode performance and
+make an informed decision about which codec and resolution to deliver to the
+user. Specifically, the API provides the developer with an estimate of
+smoothness and power efficiency of a particular codec and resolution
+combination. This allows the developer to avoid scenarios where the client is
+likely to have a poor playback experience.
+
+In Chrome, the Media Capabilities API uses metrics from previous playbacks to
+predict whether future playbacks in the same codec and at the same resolution
+will be smoothly decoded.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="./maxime-rossignol-266384-unsplash.jpg" alt="Computer motherboard near graphics card">
```

Comment:
The relevance of this image to the textual content isn't totally clear (to me). Is it possible to include an image that's more obviously related? For example, a visualization of how the Media Capabilities API establishes the threshold for when playback is considered smooth?

Reply:
This image features a Graphics Processing Unit which is actually one of the strongest signals to know a video will be decoded smoothly or not.

Reply:
True, it does include a GPU, but there's no indication of which thing in the photo is the GPU. (And the GPU isn't even in the foreground; the CPU cooler is.) But, aside from that, I'd still argue that this isn't immediately related to the text, which is about using an API to improve video playback--not about, e.g., selecting a GPU for your custom PC build. I really think it'd be a meaningful improvement if we used an image that illustrated a key concept from the text instead.

Reply:
I don't know much about hardware so the relevance of the image was a bit lost on me 😅
I think if we don't have another relevant image on hand then it's ok to remove this.

Reply:
I've removed it then.